su-template/default/layout/html/headers.erb

```diff
@@ -0,0 +1,15 @@
+  <meta charset="<%= charset %>">
+  <!-- VIEWPORT -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- AUTHOR and GENERATOR -->
+    <meta name="author" content="SketchUp Extensibility Team">
+    <meta name="generator" content="YARD <%= YARD::VERSION %> (ruby-<%= RUBY_VERSION %>) http://yardoc.org">
```

Comment:
Just realized that this line would cause all pages to change just by updating Ruby or YARD. Could cause some noisy commits which we tried to avoid earlier with that timestamp thing.